py4j-java/pom.xml

```diff
@@ -25,9 +25,9 @@
     </developer>
```

Comment:
i believe the pom.xml file is automatically created and managed by gradle, see https://github.com/py4j/py4j/blob/master/py4j-java/build.gradle#L151

changes should probably go there first?

Reply:
Oh, sure. let me take a look - actually this is my first time I use Gradle .. :-).